race.py

```diff
@@ -1,31 +1,43 @@
 import asyncio
 import time
+import operator
 
-from constants import *
-from prerace_setup import _initialize_rounds, _load_race_settings, _pre_round_preparation
+from constants import TIME_FORMAT, ORIGIN0_OFFSET
+from prerace_setup import initialize_rounds, load_race_settings, unpack_round_settings
 
 RACER_RUNTIME = 0
 RACER_NAME = 1
 
 
 def race():
```

Comment:
it looks that maybe the reason you need to pass the same data (racers, track length) every time is because race is a stateless function. Consider using objects (classes/NamedTuples etc) to pass around less useless parameters